sucks/cli.py

```diff
@@ -8,8 +8,9 @@
 import click
 from pycountry_convert import country_alpha2_to_continent_code
 
+import sucks
 from sucks import *
-
+print(sucks.__file__)
```

Comment:
Is this needed or just a forgotten `print` ?

Reply:
Just forgotten, can be omitted!